src/Config.php

```diff
@@ -19,22 +19,19 @@
  */
 class Config implements ConfigInterface
 {
-    protected $name;
-    protected $description;
-    protected $finder;
-    protected $dir;
-    protected $customFixers = array();
-    protected $usingCache = true;
-    protected $hideProgress = false;
-    protected $cacheFile = '.php_cs.cache';
-    protected $phpExecutable;
-    protected $isRiskyAllowed = false;
-    protected $rules = array('@PSR2' => true);
-
-    public function __construct($name = 'default', $description = 'A default configuration')
```

Comment:
`$description` is (and never was?) used, removing it makes the interface smaller as well